assets/js/modules/reader-revenue-manager/index.js

```diff
@@ -77,38 +84,65 @@ export const registerModule = ( modules ) => {
 };
 
 export const registerNotifications = ( notifications ) => {
-	notifications.registerNotification( 'setup-success-notification-rrm', {
-		Component: RRMSetupSuccessSubtleNotification,
-		priority: 10,
-		areaSlug: NOTIFICATION_AREAS.BANNERS_BELOW_NAV,
-		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
-		checkRequirements: async ( { select, resolveSelect } ) => {
-			const rrmConnected = await resolveSelect(
-				CORE_MODULES
-			).isModuleConnected( READER_REVENUE_MANAGER_MODULE_SLUG );
+	if ( isFeatureEnabled( 'rrmModule' ) ) {
```

Comment:
I think we can just return early here rather than wrap everything since all of these notifications would not be registered, correct? If we did have any that should still be registered, we could add them above the return instead.

Reply:
I would keep this as is only to keep things consistent as this is the pattern we are (and have been) following for all notifications and widgets registration. The additional indentation and wrapper is clearer than the "order" (before/after the feature flag return) especially if the list gets biggers.